src/apps/tabs/components/default-tab.react.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Dependencies.
+ */
+
+let React = require('react');
+let InfiniteScroll = require('react-infinite-scroll')(React);
+let moment = require('moment');
+let cache = require('lscache');
+let ProductGroup = require('./product-group.react');
+let Header = require('./header.react');
+let ProductStore = require('../../../common/stores/product');
+let api = require('../../../common/api');
+let getDay = require('../util/get-day');
+let groupByDay = require('../util/group-by-day');
+let Pane = require('../../../common/product-pane/pane.react');
+
+/**
+ * Constants.
+ */
+
+const CACHE_KEY = process.env.PRODUCTS_CACHE_KEY;
+
+/**
+ * Default Tab component.
+ *
+ * Renders the default tab app.
+ *
+ * Usage:
+ *
+ * ```js
+ * <DefaultTab />
+ * ```
+ *
+ * State:
+ *
+ * - `products`: Products to be shown on the page
+ * - `url`:      Product pane url
+ *
+ * @class
+ */
+
+let DefaultTab = React.createClass({
+
+  /**
+   * Return initial state.
+   *
+   * @returns {Object}
+   */
+
+  getInitialState() {
+    this.cache = cache.get(CACHE_KEY);
+    let startPage = !!this.cache ? 0 : -1;
```

Comment:
this is pretty much the only non-self-explaining line you have so far
could you add a short note why this is needed (i guess it's obvious) but just to make sure